pkg/controllers/scheduler/schedulingtriggers.go

```diff
@@ -276,3 +276,55 @@ func (s *Scheduler) enqueueFederatedObjectsForCluster(cluster pkgruntime.Object)
 		s.worker.EnqueueObject(fedObject)
 	}
 }
+
+// enqueueFederatedObjectsForProfile enqueues all federated objects that match the profile
```

Comment:
Why do we enqueue the objects if the trigger hash stays the same? Is it only to handle add events? If that's the case, can we only respond to add events?

Reply:
Yes, to handle add events. Since we have not decided how to respond to profile changes, we can ignore update and delete events for now

Reply:
Actually would it be better to remove all event handlers for profile until we are sure how to respond to it? cc @SOF3

Reply:
i'm going to remove the event handlers for scheduling profile until we get a better idea of how we want to deal with profile events

Reply:
Can we delete this function as well since it would be dead code

Reply:
done